conf/documentation.conf

```diff
@@ -1182,6 +1182,13 @@ description=<<EOT
 Use the RADIUS username instead of the TLS certificate common name when doing machine authentication.
 EOT
 
+[advanced.vlan_pool_technique]
```

Comment:
Missing i18n for this field

Reply:
Still missing i18n for this field

Reply:
I use to launch extract_i18n_strings.pl before each release to translate each new strings.
So do you really want me to fix it ?

Reply:
No, that's good then, I'm used to putting them.
Now I know that it was useless!
All those precious seconds I lost, I'll never get them back :cry: